os/Makefile.unix

```diff
@@ -686,3 +686,15 @@ romfs:
 ifeq ($(CONFIG_FS_ROMFS),y)
 	$(Q) ../tools/fs/mkromfsimg.sh
 endif
+
+# tools/compression/mkcompresstool.sh
+#
+# This target will make a compression tool by calling mkcompresstool.sh.
+# This compression tool (mkcompressimg) will be used to compress binaries.
+# Hence, this tool needs to be created before we build those binaries.
+
+tools/compression/mkcompresstool.sh: include/tinyara/config.h
+ifeq ($(CONFIG_COMPRESSED_BINARY),y)
+	make clean -C tools/compression/
```

Comment:
Let's add `$(Q)` at start of this line command.

Reply:
ok..  will make this change